requirements.txt

```diff
@@ -1,6 +1,6 @@
-pycaption>=0.7.1,<1.0
+le-pycaption==2.2.0a1
```

Comment:
why do we pin `le-pycaption` and `XBlock` libs?

Reply:
At the moment of upgrading the Video XBlock, these libs are compatible with others and seem to work correctly, and in order to avoid another kinds of incompatibility in the nearest future I've pinned them.